### PR TITLE
Added a new level theme superheroes

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Updated js files, added tiles
Superhero level

### CODE REVIEW

**Comment by arjunpandey2003** [comment]
collaboration @arjunpandey2003 , @samyakbvs , @saarthakchats

**Comment by ShivamRai2003** [comment]
YES YOUR TRAVIS CI BUILD IS FAILING BECAUSE OF OTHER REASON

**Comment by arjunpandey2003** [comment]
@niccokunzmann review needed

**Comment by niccokunzmann** [comment]
Hi, you added a new level, thanks for that! One thing: Where did you get the pictures from and do you have the rights to these pictures you include?

**Comment by niccokunzmann** [comment]
#416

**Comment by arjunpandey2003** [comment]
@niccokunzmann free pics which can be used commercially... advanced search on google

File: tiles/License.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+License for Superhero themed tile images
+All images taken from https://pixabay.com
+Pixabay is a vibrant community of creatives, sharing copyright free images and videos. All contents are released under Creative Commons CC0, which makes them safe to use without asking for permission or giving credit to the artist - even for commercial purposes.
+Images and Videos on Pixabay are released under Creative Commons CC0. To the extent possible under law, uploaders of Pixabay have waived their copyright and related or neighboring rights to these Images and Videos. You are free to adapt and use them for commercial purposes without attributing the original author or source. Although not required, a link back to Pixabay is appreciated.
```
Comment (niccokunzmann) [inline tiles/License.md]: I see, I will check again.

**Comment by arjunpandey2003** [comment]
@niccokunzmann yes those images had copyright issues, these don't

**Comment by niccokunzmann** [comment]
One request: For future development, could you add the tiles with the heroes you created right now to a separate folder and add the license to that? This way, if someone adds another super hero, it does not automatically get this license text. Also, not every person may know which is a superhero and which one is not.

**Comment by niccokunzmann** [comment]
What do you think?

**Comment by samyakbvs** [comment]
@niccokunzmann You mean to say that instead of tiles.js we create a new Js file

**Comment by samyakbvs** [comment]
Can we open an issue for that later and merge this rn?

**Comment by niccokunzmann** [comment]
I mean to say: please add the files not like this:

`tiles/rooms/floor/character3.svg`
but like this: 
`tiles/rooms/floor/superhero/character3.svg`

and the license file is not

` tiles/license.md`

but 

` tiles/rooms/floor/superhero/license.md`

**Comment by samyakbvs** [comment]
Oh okay

**Comment by samyakbvs** [comment]
Will do that in 5 min

**Comment by niccokunzmann** [comment]
Then, it is structurally clear where the license belongs to. Otherwise you need to know what a superhero is before you can understand the license.

**Comment by niccokunzmann** [comment]
sure, thanks!

**Comment by niccokunzmann** [comment]
One more idea: #419 adds also pictures from pixbay.
Maybe you can use the folder `pixbay` for that.

**Comment by samyakbvs** [comment]
@niccokunzmann I have done the changes as listed above now shall I make the changes again?

**Comment by samyakbvs** [comment]
Or you can merge and I will do create the pixbay folder in the next PR

**Comment by niccokunzmann** [comment]
Please see the travis build and fix it : https://travis-ci.org/fossasia/labyrinth/builds/327802028#L627

**Comment by niccokunzmann** [comment]
There are four errors.

**Comment by niccokunzmann** [comment]
Checking tiles.js
tiles.js: line 1177, col 79, Missing semicolon.
tiles.js: line 1183, col 29, Missing semicolon.
tiles.js: line 1190, col 89, Missing semicolon.
tiles.js: line 1198, col 9, Duplicate key 'both'.

**Comment by arjunpandey2003** [comment]
completing

**Comment by niccokunzmann** [comment]
Please see the build. I am here to answer questions.

**Comment by samyakbvs** [comment]
tiles.js: line 1198, col 9, Duplicate key 'both'.

**Comment by samyakbvs** [comment]
What do you mean by this?

**Comment by arjunpandey2003** [comment]
@samyakbvs fixing

**Comment by arjunpandey2003** [comment]
@niccokunzmann i think most of the things are done can you help us in travis

**Comment by samyakbvs** [comment]
@niccokunzmann pls merge

**Comment by arjunpandey2003** [comment]
@niccokunzmann will keep in mind sorry

**Comment by niccokunzmann** [comment]
Thanks!

**Comment by niccokunzmann** [comment]
Cool, looks like a nice cooperative effort. What did you like about it? Where did you see challenges cooperating?

**Comment by arjunpandey2003** [comment]
@niccokunzmann copyrighting issues were a challenge and proof reading big files, i like tha we finally get a substantial contribution to labyrinth!!!

**Comment by samyakbvs** [comment]
Ya we were trying to make a level from so long

**Comment by niccokunzmann** [comment]
You succeeded 🎆

**Comment by arjunpandey2003** [comment]
@niccokunzmann thanks!!!!

**Comment by samyakbvs** [comment]
But how can we see our level @niccokunzmann

**Comment by niccokunzmann** [comment]
You can create a test link where your level is loaded instead of the other level. You can change gui.js for that. It should be a one-liner, exchanging the function.

**Pull request merged**